contextutil/context.go

```diff
@@ -48,8 +48,8 @@ func Tenant(ctx context.Context) (config.TenantConfiguration, error) {
 	return tc, nil
 }
 
-// CentContext updates a context with tenant info using the configstore, must only be used for api handlers
-func CentContext(ctx context.Context, cs config.Service) (context.Context, error) {
+// NodeContext updates a context with tenant info using the configstore, must only be used for api handlers
+func NodeContext(ctx context.Context, cs config.Service) (context.Context, error) {
 	tcIDHex := ctx.Value(config.TenantKey).(string)
```

Comment:
Can you create here the wrapping function you said you were going to do earlier?

Reply:
I think I didn't agree with that change since its not something that should be used commonly in the node.